content/gamenews

```diff
@@ -1,4 +1,6 @@
-2015-12-24 12-00-00:
-	Title: Happy Holidays! The release is out!
-	Author: Oliver Brakmann
-	Content: For Christmas, the OpenRA developers give you a brand-new shiny OpenRA release!\n\nIt has a new global chat, re-designed server browser and lobby, "joystick" scrolling, Defend stance by default for units, a completely re-balanced Dune 2000 mod, improved balancing, bugfixes and many things more!\n\nHead over to our website at http://openra.net to read the full release announcement, and see the complete unabridged list of new features, improvements and fixes at http://changelog.openra.net.\n\nHave fun, and Merry Christmas to all!
+2016-02-21 12-00-00:
+	Title: The March 2016 Tournament Is Coming!
+	Author: Krystal Amaia
+	Content: There is a brand new OPENRA TD/RA/D2K Tournament Coming in March 2016!  For Complete Details see http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&t=19415
```

Comment:
I love it!
OPENRA should be OpenRA and For Complete Details should be For complete details